min.py

```diff
@@ -135,6 +136,37 @@ def get_min_scipy_version(min_python, min_numpy):
     return df.SciPy_version
 
 
+def find_pkg_mismatches(pkgs, fnames):
```

Comment:
So, as mentioned before, I think it is better to make this function:

```
def find_pkg_mismatches(pkg_name, pkg_version, fnames):
```

rather than doing ALL packages at the same time. The code that I shared yesterday was purely for demo purposes and was not intended to suggest that we should do all packages at the same time so apologies for that confusion.